composer.json

```diff
@@ -20,8 +20,8 @@
     "require": {
         "php": "^7.1",
         "psr/http-message": "^1.0",
-        "psr/http-client": "^0.1",
-        "php-http/httplug": "^1.1 || ^2.0@dev",
+        "psr/http-client": "^0.3",
+        "php-http/httplug": "^1.1 || 2.0.*",
```

Comment:
why `2.0.*` and not `^2.0`. It `2.0.*` is working, next 2.x versions will work too (unless you stop following semver in httplug, but please don't do that)

Reply:
I have a hard time making the branch alias work for HTTPlug. We got a 2.x branch with branch alias: https://github.com/php-http/httplug/blob/2.x/composer.json

I read [composer docs](https://getcomposer.org/doc/articles/aliases.md) and I thought I could do like this...

The current status is: Travis will not install php-http/httplug 2.x.